### DISCUSSION TITLE
Error running script

### DISCUSSION THREAD

**Discussion by nicowagner89** [HotCakeX/Harden-Windows-Security#182] [Q&A] [answered]
I'm running **irm 'https://raw.githubusercontent.com/HotCakeX/Harden-Windows-Security/main/Harden-Windows-Security.ps1' | iex** on Powershell 7.4.0 as administrator after confirming I meet all the requirements and am getting error "Line 2398 | Protect-WindowsSecurity Cannot bind argument to parameter 'Path' because it is null".

**Comment by HotCakeX** [comment, marked as answer]
Hi,
I just pushed an update, please try again and let me know if you're still getting that error.

**Reply by nicowagner89** [reply]
Yes, I can run it now. Thanks for your help!

**Reply by HotCakeX** [reply]
That's great, you're welcome!
